### DISCUSSION TITLE
Add layout without message bubbles to ElementX

### DISCUSSION THREAD

**Discussion by Reinachan** [element-hq/element-meta#2276] [Ideas]
I'm not a big fan of message bubbles. I'm probably far from the only person to dislike message bubbles, especially with the popularity of chatting services like Discord and Slack. Bubbles also don't make as much sense in group chats because in groups chats there are more than just you and a recipient, meaning the distinction between you and the other person through bubble colour and text alignment makes even less sense.

I'm not suggesting that bubbles should be removed btw. But I'm suggesting that an optional chat layout without them is added like in old Element. Writing out my reasoning above, I even think it would be cool if you could independently choose between the two layouts based on whether the current room is considered a DM or a group chat.

**Comment by o0oo00o0o** [comment]
This was an option on Element X as recently as March 2024. The option to not use chat bubbles is the only reason I use Element over other Matrix clients. I don't know who needs to see this, but bring this option back, please.

**Reply by Reinachan** [reply]
It was apparently only an option on iOS. Iirc, when they removed it from there, they said they had plans to bring it back eventually
